### ISSUE TITLE
[LINT_BUG]: Linter does not accept attached list from package

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#88]
### box.linters version

0.9.0

### Sample source code to lint

```R
box::use(
  rhino[log, react_component],
  shiny,
)

box::use(app/view/hello, )

#' @export
server <- function(id) {
  shiny$moduleServer(id, function(input, output, session) {
    log$trace("This is a test")
    hello$server("hello")
  })
}
```

### Lint command used

```
lintr::lint(
  code,
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)
```

### Lint result

```
Warning: file=app/main.R,line=2,col=9,[box_unused_attached_pkg_fun_linter] Imported function unused.
```

Line 2, col 9 -> `log`

### Expected result

This should not lint
